.travis.yml

```diff
@@ -41,6 +41,10 @@ matrix:
       env: JOB=babylon-flow-tests
     - node_js: "node"
       env: JOB=babylon-test262-tests
+    - os: osx
+      node_js: "node"
+  allow_failures:
+    - os: osx
```

Comment:
I made it optional so it doesn't impact the build time, we will an email for failure anyway.